modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMvPartitionStorage.java

```diff
@@ -627,10 +752,14 @@ public void forEach(BiConsumer<RowId, BinaryRow> consumer) {
 
     /**
      * Deletes partition data from the storage.
+     *
+     * @param writeBatch Write batch to add delete requests to.
      */
-    public void destroy() {
+    public void destroy(WriteBatch writeBatch) {
```

Comment:
I'm not sure, but wouldn't it be better to remove the partition id from the meta CF here as well? This way we will not have to accept a `WriteBatch`

Reply:
Sure